### PR TITLE
tnoodle-ui - Updating Project to Create React App 2.0

### PR DESCRIPTION
**Pull request by olezhek85**
- Updated react-scripts to "2.0.5" version
- Added rel="noopener noreferrer" to target blank links
- Formatting code

### CODE REVIEW

**Comment by jfly** [comment]
Thanks, dude! Could you split this up into 3 PRs (or 1 PR with 3 commits)? It's hard to tell what changed because of all the code reformatting.

**Comment by olezhek85** [comment]
@jfly , done!

File: tnoodle-ui/src/Layout.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ function analyzeVersion(versionInfo) {
   }
   var isAllowed = versionInfo.allowed.indexOf(runningVersion) >= 0;
   var isCurrent = runningVersion === versionInfo.current.name;
-  var downloadCurrent = (<span><a href={versionInfo.current.download} target='_blank'>{versionInfo.current.name}</a> (<a href={versionInfo.current.information} target='_blank'>details</a>)</span>);
+  var downloadCurrent = (<span><a href={versionInfo.current.download} target='_blank' rel="noopener noreferrer">{versionInfo.current.name}</a> (<a href={versionInfo.current.information} target='_blank' rel="noopener noreferrer">details</a>)</span>);
```
Comment (jfly) [inline tnoodle-ui/src/Layout.js]: > Added rel noopener and noreferrer to target blank links

This commit message tells me what you did, but not *why* you did it. I can figure out what you did by looking at the diff =) I had to do some googling to figure out why we want to do this. Could you update your commit message to explain why you did what you did, not just what you did?

Also, how do you feel about creating an `ExternalLink` component that we could use across all of TNoodle? That would also give us a nice place to add a comment explaining why we want this `noopener noreferrer` stuff.

**Comment by jfly** [comment]
It looks like the build might be failing related to your changes? See https://travis-ci.org/thewca/tnoodle/builds/444150097#L1641-L1642:

```
[1/4] Resolving packages...
-walk@^0.1.0inimalistic-crypto-utils@^1.0.1[2/4] Fetching packages...
error css-loader@1.0.0: The engine "node" is incompatible with this module. Expected version ">= 6.9.0 <7.0.0 || >= 8.9.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I imagine this is some crazy fallout from upgrading react-scripts? Unfortunately, we'll need to figure this out before we can merge this up.

**Comment by jfly** [comment]
Closing due to inactivity.

**Pull request closed without merging**